### PR TITLE
Html personal website

### PR DESCRIPTION
**Pull request by LuamB**
Created personal website with navigation bar, image and link.

### CODE REVIEW

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  </head>
+  <body>
+    <!--[if lt IE 7]>
+      <p class="browsehappy">
+        You are using an <strong>outdated</strong> browser. Please
+        <a href="#">upgrade your browser</a> to improve your experience.
+      </p>
+    <![endif]-->
+    <header>
+      <nav>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:23]: you could/should add ul and li elements to your nav

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      <p>
+        Luam is a people person and communication expert with Psychology
+        background (Research Master, major: Cognitive Neuroscience). She has a
+        passion for both human and programming languages. Words and code have
+        the power to affect change and she will be part of it.
+      </p>
+    </section>
+
+    <section id="Projects">
+      <p>
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:46]: I think it doesn't need the p here at this point. You could remove it to keep the code clean, but you don' have to :)

File: html-and-web/html-and-the-web_personal-website/index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      </p>
+    <![endif]-->
+    <header>
+      <nav>
+        <a href="#Biography">Biography</a>
+        &nbsp;
+        <a href="#Contact">Contact</a>
+      </nav>
+    </header>
+
```
Comment (itsTHL) [inline html-and-web/html-and-the-web_personal-website/index.html:29]: you could add the main-element here to show where the main content of your page is

Comment (LuamB) [reply html-and-web/html-and-the-web_personal-website/index.html:31]: Yes, I will 👍🏾

**Pull request merged**